routers/web/user/auth.go

```diff
@@ -649,14 +649,28 @@ func SignInOAuthCallback(ctx *context.Context) {
 				ctx.ServerError("CreateUser", err)
 				return
 			}
+
+			oauthProviderName := strings.ToLower(provider)
```

Comment:
What does this actually? 

This adds the name of the oauth into a new database table, link within the user to the ID of that oauth. But that doesn't solve the issue of adding a suffix.

Also, we already have this, look at the `LoginSource` field ;) which should give you the same way of getting the Oauth's name.